hawk/src/test/java/com/orhanobut/hawk/HawkTest.java

```diff
@@ -44,7 +46,7 @@ public HawkTest() {
     context = Robolectric.buildActivity(Activity.class).create().get();
   }
 
-  @Override public void setUp() throws Exception {
+  @Override @Before public void setUp(){
```

Comment:
Before and After annotations are redundant here since test class is extended from testcase, I suggest to remove inheritance and keep using annotations here.